scripts/code_coverage_report/code_coverage_file_list.json

```diff
@@ -5,7 +5,7 @@
       "FirebaseABTesting.*",
       "Interop/Analytics/Public/[^/]+\\.h",
       "\\.github/workflows/abtesting\\.yml",
-      "Gemfile"
+      "^Gemfile"
```

Comment:
Update the pattern to avoid file path including `Gemfile` to trigger workflows.

Reply:
Why not delete the line?

Reply:
The patterns were following the pull_request path in sdk workflows, e.g. [database](https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/database.yml#L11). Will remove it since it is not related to code coverage.